### ISSUE TITLE
Wrong solution for collection.P05

### ISSUE THREAD

**Issue by dominik223** [jaceklaskowski/scalania#35]
The given solution can't handle situation when List is made from only one element, list contains elements that are not repeated, or if the last element is not repeated.

**Comment by jaceklaskowski** [comment]
Please write a test to prove your thinking (using ScalaCheck).

**Comment by jaceklaskowski** [comment]
Is this still a valid issue?

**Comment by dominik223** [comment]
Yes it is! My test proof that current solution is wrong. So issue is still valid.
